src/unix/darwin.c

```diff
@@ -333,3 +333,29 @@ void uv_free_interface_addresses(uv_interface_address_t* addresses,
 
   uv__free(addresses);
 }
+
+int uv__tcp_enable_largesocket(uv_tcp_t *tcp) {
+  int rmem;
+  int wmem;
+  int r;
+
+  size_t size = sizeof(int);
+  if (sysctlbyname("net.inet.tcp.recvspace", &rmem, &size, NULL, 0) ||
```

Comment:
the 3rd param is an IN/OUT variable which supplies the size of the buffer before the call, and returns the size after the call and it is an address type and I guess it requires an lvalue which sizeof cannot provide?